build.gradle

```diff
@@ -31,6 +31,7 @@ buildscript {
 // ktlint Plugin
 plugins {
     id "org.jlleitschuh.gradle.ktlint" version "10.2.1"
+    id 'com.autonomousapps.dependency-analysis' version "1.0.0"
```

Comment:
After a discussion with the team, wwe are wondering of the overhead of adding a generally unused plugin. We think this is still acceptable.

Also maybe we could try to add a CI check using this plugin, and ensure it fails when unused dependencies are detected, ignoring false positive.

Reply:
The full dependency analysis takes a long time, so I think it's a better idea to run it in parallel to all the other checks, but let me know if it's not feasible.
Here is an example workflow run: https://github.com/vector-im/element-android/runs/6505420170?check_suite_focus=true